azure-devops/Create-PRDiff.ps1

```diff
@@ -0,0 +1,19 @@
+[CmdletBinding(PositionalBinding=$False)]

```

Comment:
Should this be:

```ps1
# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception OR MIT
```

since it's copied from vcpkg, which is under the MIT license?